samples/05-file-transfer-cloud/consumer/build.gradle.kts

```diff
@@ -42,11 +41,9 @@ dependencies {
     implementation(project(":extensions:api:data-management"))
 
     implementation(project(":data-protocols:ids")) {
-        exclude("org.eclipse.dataspaceconnector","ids-token-validation")
+        exclude("org.eclipse.dataspaceconnector", "ids-token-validation")
```

Comment:
this exclusion is not needed anymore, as `ids-token-validation` was removed from `data-protocols:ids` BOM